main.py

```diff
@@ -144,10 +147,40 @@ def check_date(store, date_to_check):
         '{}: {} -> {}'.format(op, last[astart:aend], booked_list[bstart:bend])
         for op, astart, aend, bstart, bend in  difflib.SequenceMatcher(None, last, booked_list).get_opcodes()
         if op != 'equal'
-    ]
+    ] #Contains the difference between the current details and the previous history file.
 
     if diff:
         pprint(diff)
+        send_email(diff)
+
+def send_email(diff):
+    email_body = str(diff).replace('–','-').replace('\', \'', '\',\n\'') #Changing the dashes is more legible from terminal, can remove that for real emails.
+    sent_from = 'sender@obsi.com.au'
+    to = ['robinchew@gmail.com']
+    gmail_user='mb36340'
+    gmail_password = ''
+
+    subject = 'New booking details'
+    email_text = f"""\
+    Message-ID: <{uuid4()}@obsi.com.au>
```

Comment:
environment variable for the domain part 'obsi.com.au'. Can be extracted from the FROM environment variable